.drone.yml

```diff
@@ -172,6 +172,25 @@ steps:
       - master
       - "*-stable"
 
+# http://plugins.drone.io/drone-plugins/drone-slack/
+- name: Slack Notify
+  image: plugins/slack
+  settings:
+    webhook:
+      from_secret: slack_webhook
+    channel:
+      from_secret: slack_webhook_channel
+    template: >
+      *{{uppercasefirst repo.name}}*
+      {{#success build.status}}
+        [SUCCESS] Branch {{ build.branch }} by {{ build.author }}. Good job.
```

Comment:
we dont need this step
we have sdk:postbuild rake task where we configure slack channel